src/main/java/io/opentracing/contrib/concurrent/TracedCallable.java

```diff
@@ -11,20 +12,24 @@
 public class TracedCallable<V> implements Callable<V> {
 
   private final Callable<V> delegate;
-  private final Continuation continuation;
+  private final Span continuation;
+  private final Tracer tracer;
 
-  public TracedCallable(Callable<V> delegate, ActiveSpan activeSpan) {
+  public TracedCallable(Callable<V> delegate, Tracer tracer) {
     this.delegate = delegate;
-    this.continuation = activeSpan != null ? activeSpan.capture() : NoopContinuation.INSTANCE;
+    this.tracer = tracer;
+    this.continuation = tracer.scopeManager().active() == null ? null : tracer.scopeManager().active().span();
```

Comment:
I think it's cleaner to use `NoopScope` like it was before.

Reply:
Its not the scope that is being passed across threads, it is the span.

Reply:
could you rename continuation then to something more accurate